File: Day-07/FahrenheitToCelsius.py
def FahrenheitToCelsius(n):
    i = 0;
    while(i <= n):
        celsius = ((i - 32)*5)/9
        print('{} Fahrenheit = {} Celsius'.format(i, celsius) )
        i = i + 1

FahrenheitToCelsius(100)

File: Day-07/FahrenheitToCelsius.js
function fahrenheitToCelsius(n) {
  let i = 0;
  while (i <= n) {
    const celsius = ((i - 32) * 5) / 9;
    console.log(`${i} Fahrenheit = ${celsius} Celsius`);
    i++;
  }
}

fahrenheitToCelsius(100);
